sagemaker-pipeline-multi-model/restate-project.ipynb

```diff
@@ -2,31 +2,31 @@
  "cells": [
```

Comment:
<p>Does this notebook need conda/mxnet, or is a basic conda_python3 kernel sufficient?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3488/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
The pipeline specifically requires python 3.7, hence the recommendation of kernel conda_mxnet_p37. conda_python3 is using python 3.8. When I attempt to build my pipeline using conda_python3, I encounter the following error:
```
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
daal4py 2021.3.0 requires daal==2021.2.3, which is not installed.
sagemaker-pyspark 1.4.2 requires pyspark==2.4.0, but you have pyspark 3.0.0 which is incompatible.
numba 0.54.1 requires numpy<1.21,>=1.17, but you have numpy 1.23.0 which is incompatible.
awscli 1.22.97 requires botocore==1.24.42, but you have botocore 1.24.46 which is incompatible.
aiobotocore 2.0.1 requires botocore<1.22.9,>=1.22.8, but you have botocore 1.24.46 which is incompatible.
```
Thank you!